README.md

```diff
@@ -1,5 +1,11 @@
 # Smart Contract init
-Service Will initialize Streamrs Smart Contracts
+Service Will initialize Streamr's Smart Contracts
```

Comment:
What does this mean? Is it a title or a documentation TODO item?

Reply:
I'll add more description. This was Mikhael's text